clang/include/clang/CConv/ConstraintsGraph.h

```diff
@@ -86,10 +85,10 @@ class ConstraintsGraph
     return !Atoms.empty();
   }
 
+  vertex_t addVertex(Atom *A);
```

Comment:
Is this dangerous? You have a virtual method by the same name, don't you?

Reply:
No. The superclass made it protected. However, the subclass can make the function public, a trick to allow subclass to expose parent class methods.